src/tbb/governor.h

```diff
@@ -148,6 +155,70 @@ class governor {
         return false;
 #endif
     }
+
+#if __TBB_STATISTICS
+    enum class statistics : std::size_t {
+        executed = 0,
+        try_get,
+        got,
+        try_steal,
+        stolen,
+        empty_vic_pool,
+        size,
+    };
+    using array_stat = std::array<std::uint32_t, static_cast<std::size_t>(statistics::size)>;
```

Comment:
What about using `std::map<statistics, std::size_t>` instead of `std::array` to store results?

Reply:
By they are exactly the same by API, but array has no extra allocations, so statistics will not slow the program and will be more correct